instrumentation/opentelemetry-api-metrics-1.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/metrics/MeterTest.java

```diff
@@ -43,16 +43,15 @@
   @BeforeEach
   void setupMeter(TestInfo test) {
     instrumentationName = "test-" + test.getDisplayName();
-    meter = GlobalMeterProvider.getMeter(instrumentationName, "1.2.3");
+    meter = GlobalMeterProvider.get().get(instrumentationName, "1.2.3", "http://schema.org");
   }
 
   @Test
   void longCounter() {
-    LongCounter instrument =
-        meter.longCounterBuilder("test").setDescription("d").setUnit("u").build();
+    LongCounter instrument = meter.counterBuilder("test").setDescription("d").setUnit("u").build();
```

Comment:
@jsuereth Can you just go through MeterTest to verify I migrated from old to new correctly?

Reply:
Hey, sorry, I was out sick two days ago, just getting to this now.